themes/aquablank/layouts/partials/startnow.html

```diff
@@ -0,0 +1,13 @@
+
+                <div class="avdcve_sidebar_wrap ">
+                    <div class="project_item">
+                        <h2 class="is-size-4 has-text-weight-medium my-2">start protecting your cloud</h2>
+                        <p class="my-2">
+                            <a href="#" target="_blank" class="button is-primary are-large is-large is-fullwidth">Start Now</a>
+					
+                        </p>
+
+                       
+                    </div>
+                   
+                </div>
```

Comment:
Is this the right size? Looks quite off for me: 
<img width="1460" alt="image" src="https://user-images.githubusercontent.com/1254783/125675075-03952b3d-a409-4bcb-809a-1184b1e62ef2.png">